tests/Fist/regtest-spgr/TEST_FILES

```diff
@@ -0,0 +1,10 @@
+# runs are executed in the same order as in this file
+# the second field tells which test should be run in order to compare with the last available output
+# e.g. 0 means do not compare anything, running is enough
+#      1 compares with the space group name and number of operations provied in the SPGR section of the output file
+#      for details see cp2k/tools/do_regtest
+copper.inp                                   1      Fm-3m   192
+quartz.inp                                   1      P3_221    6
+zro.inp                                      1      P2_1/c    4
+zro-filename.inp                             1      P2_1/c    4
```

Comment:
The last two columns look strange. They should be the threshold, e.g. `1e-14`, and the reference results, e.g. `-0.211229963864E+00`.

Reply:
These tests are to make sure space group symmetry works. The final energy does not really matter.

Reply:
If possible you should use the opportunity to also compare something from the output. It makes for a much better test.


If you really just want to check that it doesn't crash then you have to use test type zero.
So, your `TEST_FILES` should look like this:
```
# you can add comments like this
copper.inp                                   0
quartz.inp                                   0
# comment lines can be anywhere
zro.inp                                      0
zro-filename.inp                             0
```